lib/screens/quick_rules_page.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/material.dart';
+
+class QuickRulesPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(
+          title: Text('Quick Rules'),
+        ),
+        body: Container(
+            padding: const EdgeInsets.all(16.0),
+            child:
+                Column(crossAxisAlignment: CrossAxisAlignment.start, children: [
+              Text(
```

Comment:
To me this way of doing it looks ugly, but because I wanted to embed the icons into the text this was the best I could come up with, open to other ideas.

Reply:
I can't think of a better way, although the indentation here looks a bit funky